frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,13 @@
+import { type IconName, iconNameToIcon } from './common.js';
+
+type Properties = {
+  iconName: IconName;
```

Comment:
Let's also add optional `className` prop for icons
```suggestion
  className?: string;
  iconName: IconName;
```